OasisDEX/oasis-borrow
# Ades/aave/update proxy copy

# [Update proxy copy in UI](https://app.shortcut.com/oazo-apps/story/6834/oasis-borrow-update-proxy-copy-in-ui)
  
## Changes 👷‍♀️
- remove references to only one proxy being necessary in the UI
  
## How to test 🧪
  <Please explain how to test your changes>
- go to the proxy steps in DS proxy products and check that their is no reference to the proxy only being needed once
